deliverytiersuite/delivery-tier-suite/metadata/src/main/resources/changeLog.xml

```diff
@@ -17,9 +17,7 @@ again in database then this changeset will not run until we change the changeset
             <sqlCheck expectedResult="0">SELECT count(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_NAME = 'blog_post_visit' AND COLUMN_NAME ='VISIT_ID' AND DATA_TYPE = 'bigint' AND COLUMN_DEFAULT IS NULL AND IS_NULLABLE = 'NO' AND EXTRA like '%auto_increment%'
```

Comment:
@rgarg24  I notice that the table and data type are lowercase here, is that an issue?

Reply:
> @rgarg24 I notice that the table and data type are lowercase here, is that an issue?

Nate, that is not an issue. This query runs in linux as well because here 'blog_post_visit' and 'bigint' are values corresponding to columns.
case sensitivity is there in case of table names and database names.